tests/unit_tests/commandline/test_log_setup.py

```diff
@@ -23,23 +22,53 @@ def generate_random_string(length):
     return "".join(random.choices(string.ascii_letters, k=length))
 
 
+def insert_word_like_spaces_to_string(string):
+    """Modify the input string by inserting spaces to make "words" of length 2-8.
+
+    Parameters
+    ----------
+    str (str):
+        The input string to be modified.
+
+    Returns
+    -------
+    str (str):
+        The modified string with spaces inserted at random locations.
+
+    Example:
+    >>> insert_word_like_spaces_to_string("HelloWorld")
+    'He lloW or ld'
+    """
+    loc = random.randint(2, 3)
+    while loc < len(string):
+        string = string[:loc] + " " + string[loc + 1 :]
+        loc += random.randint(2, 8)
+    return string
+
+
 def generate_random_messages():
     """Generate a random amount of messages with random length."""
-    num_messages = random.randint(1, 50)
-    return [generate_random_string(random.randint(5, 110)) for _ in range(num_messages)]
+    num_messages = 20
```

Comment:
Thank you for making this a fixed number of tests. That will fix a lot of confusion.

Is there any reason that 20 wouldn't be a sufficient number of tests?

Reply:
Okay, I think this sounds fine to me. If we start having random failures in this, though, we may want to specifically add a few of the edge cases. Things like specific special characters that we expect could cause problems.

I don't see any reason that we would actually expect to have issues with any edge cases, though.